.circleci/config.yml

```diff
@@ -236,9 +236,18 @@ jobs:
           environment:
             ENTERPRISE: 1
           command: mix test.integration_block_submission_vault
-      # - run: rm -rf deps/ _build/
-      # - run: ENTERPRISE=0 mix deps.get
-      # - run: mix test.integration_block_submission
+      - run: rm -rf deps/ _build/
```

Comment:
maybe to run `ENTERPRISE: 0` as a separate step?

Reply:
step? do you mean job?

Reply:
yes